### PR TITLE
adding few missing wrappers

### PR DESCRIPTION
**Pull request by adizaimi**
adding few missing wrappers

### CODE REVIEW

File: berkdb/btree/bt_search.c [hunk 1]
```diff
@@ -227,7 +227,7 @@ genid_hash_init(DB_ENV *dbenv, int szkb)
 		}
 	}
 
-	pthread_mutex_init(&(h->mutex), NULL);
+	 pthread_mutex_init(&(h->mutex), NULL);
```
Comment (akshatsikarwar) [inline berkdb/btree/bt_search.c]: ?

Comment (adizaimi) [reply berkdb/btree/bt_search.c]: ? ?

**Comment by adizaimi** [comment]
No new crashes on IBM.

File: berkdb/btree/bt_search.c (hunk 1 above)
Comment (adizaimi) [reply berkdb/btree/bt_search.c]: Was to test the warning above. It works.

Comment (akshatsikarwar) [reply berkdb/btree/bt_search.c]: IC

Comment (akshatsikarwar) [reply berkdb/btree/bt_search.c]: THX

**Pull request merged**